include/deal.II/lac/cuda_sparse_matrix.h

```diff
@@ -136,6 +150,51 @@ namespace CUDAWrappers
      */
     std::size_t
     n_nonzero_elements() const;
+
+    /**
+     * Print the matrix to the given stream, using the format <tt>(row,column)
+     * value</tt>, i.e. one nonzero entry of the matrix per line. If
+     * <tt>across</tt> is true, print all entries on a single line, using the
+     * format row,column:value.
+     *
+     * If the argument <tt>diagonal_first</tt> is true, diagonal elements of
+     * quadratic matrices are printed first in their row, corresponding to the
+     * internal storage scheme. If it is false, the elements in a row are
```

Comment:
This is not how it is stored internally. The deal.II matrix is reordered to match cuSPARSE format.